### PR TITLE
Remove space buttons

### PR DESCRIPTION
**Pull request by djangoprince**

### CODE REVIEW

**Comment by djangoprince** [comment]
play the right clip when skipping sentences

File: src/components/contribute/carousel/wheel.tsx [hunk 1]
```diff
@@ -98,7 +96,6 @@ interface State {
     audioError?: AudioError;
     uploadError?: UploadError;
     expanded: boolean;
-    outOfClips?: boolean;
```
Comment (staffanru) [inline src/components/contribute/carousel/wheel.tsx:101]: why is this removed?

File: src/components/contribute/carousel/wheel.tsx [hunk 2]
```diff
@@ -666,8 +645,10 @@ class CarouselWheel extends React.Component<Props, State> {
             contribute: { expanded, gaming, goal, progress },
         } = this.props;
         const activeClip = clips[clipIndex] || undefined;
-        const activeClipToRepeat =
-            (clipsToRepeat && clipsToRepeat[clipIndex]) || undefined;
+        //return the clip from non skeipped sentences in getActualClipToRepeat function
+        const activeClipToRepeat = this.getActualClipToRepeat();
+        console.log(activeClipToRepeat);
```
Comment (staffanru) [inline src/components/contribute/carousel/wheel.tsx:650]: we should remove the comment

File: src/components/contribute/carousel/wheel.tsx [hunk 3]
```diff
@@ -199,27 +185,20 @@ class CarouselWheel extends React.Component<Props, State> {
...
-                sentences: newSentences,
-            });
         }
+        // refresh sentences for the clips
+        const newSentences = await this.sentencesFromClips(freshClipsToRepeat);
+
+        this.setState({
+            clipsToRepeat: freshClipsToRepeat,
+            sentences: newSentences,
+        });
```
Comment (staffanru) [inline src/components/contribute/carousel/wheel.tsx:201]: I think all of these changes should be reverted back? Did you make them?

File: src/components/contribute/carousel/wheel.tsx [hunk 4]
```diff
@@ -127,7 +124,6 @@ class CarouselWheel extends React.Component<Props, State> {
             uploadError: undefined,
 
             expanded: false,
-            outOfClips: false,
```
Comment (staffanru) [inline src/components/contribute/carousel/wheel.tsx]: This should not be removed

File: src/components/contribute/carousel/wheel.tsx [hunk 5]
```diff
@@ -24,7 +24,6 @@ import {
     decrementProgress,
     incrementProgress,
     setExpanded,
-    setGoal,
```
Comment (staffanru) [inline src/components/contribute/carousel/wheel.tsx]: Set goal is necessary

File: src/components/contribute/carousel/wheel.tsx [hunk 6]
```diff
@@ -71,7 +70,6 @@ const dispatchProps = {
...
```
Comment (staffanru) [inline src/components/contribute/carousel/wheel.tsx]: set goal is necessary

**Review by staffanru — APPROVED** [review]
